pom.xml

```diff
@@ -44,6 +44,8 @@
   <properties>
     <hadoop.version>2.6.0-cdh5.5.0</hadoop.version>
     <spark.version>1.5.0-cdh5.5.0</spark.version>
+    <hadoop.scope>compile</hadoop.scope>
+    <spark.scope>compile</spark.scope>
```

Comment:
You have to remove all the references you've added too, obviously. The code obviously builds without the code you're adding, since all the builds are green.